check_version.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+# this is a little fragile because it assumes the [package] section will be at the
+# top of the Cargo.toml
+BINARY_VERSION_STRING=$(cat steward/Cargo.toml | grep "^version.*" | head -1)
+# above output is 'version = "x.x.x"' so we trim the leading and trailing characters
```

Comment:
This would likely be more robust if we use a standard semver regex with a capture group for the version string. Not sure if this accounts for trailing elements, etc.

Reply:
I can do that; it will still capture any other toml tables (sections) that have a version line, such as `[dependencies.abscissa_core]` in the steward toml. Are we concerned with that right now? We typically keep the package section at the top.

Reply:
It's probably fine, considering this is a workflow for this specific repo and we shouldn't have expectations of that shifting out from under us. In the more paranoid case if you just wanted to stick to regex you could look for the `[package]` header as part of your pattern matching.